dt-core/global-functions.php

```diff
@@ -905,6 +905,97 @@ function dt_validate_date( string $date, string $format = 'Y-m-d H:i:s' ): bool
         }
     }
 
+    /**
+     * Dump and die
+     */
+    if ( !function_exists( 'dd' ) ) {
```

Comment:
It stands for dump and die. It dumps a given variable (or variables) and ends execution of the script. It's a convenience utility function you'll find in some PHP frameworks. I'm happy to remove it, but it's so much easier to write: 

```
dd($foo, $bar); 
```

instead of: 

```
var_dump($foo); var_dump($bar); exit;
```